src/modules/packages/main.py

```diff
@@ -337,6 +337,22 @@ def update_system(self):
         # Doesn't need to update the system explicitly
         pass
 
+class PMFlatpak(PackageManager):
+    backend = "flatpak"
+
+    def install(self, pkgs, from_local=False):
+        check_target_env_call(["flatpak", "install", "--assumeyes"] + pkgs)
+
+    def remove(self, pkgs):
+        check_target_env_call(["flatpak", "uninstall", "--noninteractive"] + pkgs)
+        check_target_env_call(["flatpak", "uninstall", "--noninteractive", "--unused"])
```

Comment:
I don't think we should be forcibly removing unused.  For several reasons:
* It doesn't seem like something an installer should be doing
* Newer versions of flatpak do this automatically making this redundant
* Depending on the situation, this might get called for every package